handsontable/src/dataMap/metaManager/metaSchema.js

```diff
@@ -1211,18 +1211,19 @@ export default () => {
-     * | `copyColumnHeaders`      | Boolean (default: `false`)                         | `true`: add the option to copy cells along with their nearest column headers                                                                                                            |
-     * | `copyColumnGroupHeaders` | Boolean (default: `false`)                         | `true`: add the option to copy cells along with all their related columns headers                                                                                                       |
-     * | `copyColumnHeadersOnly`  | Boolean (default: `false`)                         | `true`: add the option to copy just column headers (without copying cells)                                                                                                              |
-     * | `uiContainer`            | An HTML element                                    | A UI container for the secondary focusable element                                                                                                                                      |
+     * | Option                   | Possible settings                                  | Description                                                                                                                                                                                         |
+     * | ------------------------ | -------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+     * | `columnsLimit`           | A number (default: `Infinity`)                     | The maximum number of columns that can be copied                                                                                                                                                    |
+     * | `rowsLimit`              | A number (default: `Infinity`)                     | The maximum number of columns that can be copied                                                                                                                                                    |
+     * | `pasteMode`              | `'overwrite'` \| `'shift_down'` \| `'shift_right'` | When pasting:<br>`'overwrite'`: overwrite the currently-selected cells<br>`'shift_down'`: move the currently-selected cells down<br>`'shift_right'`: move the currently-selected cells to the right |
+     * | `copyColumnHeaders`      | Boolean (default: `false`)                         | `true`: add the option to copy cells along with their nearest column headers                                                                                                                        |
```

Comment:
WDYT about rephrasing descriptions to emphasize that they refer to the context menu? Something like: `true`: extend the context menu for an option that allows copy cells along with their nearest column headers.

Reply:
Sure, done: [dd8ab83](https://github.com/handsontable/handsontable/pull/10118/commits/dd8ab8380c1554f3ff6339aaa7a77c2826ca0828)